### PR TITLE
OC-254 messaging config

### PR DESCRIPTION
**Pull request by rudidevries**

### CODE REVIEW

**Comment by villgust** [comment]
please retest this

File: osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java [hunk 1]
```diff
@@ -134,278 +47,70 @@ public MessagingConfig() {
...
-    public ActiveMQConnectionFactory connectionFactory() {
-        final ActiveMQConnectionFactory activeMQConnectionFactory = new ActiveMQConnectionFactory();
-        activeMQConnectionFactory.setRedeliveryPolicyMap(this.redeliveryPolicyMap());
-        activeMQConnectionFactory.setBrokerURL(this.environment
-                .getRequiredProperty(PROPERTY_NAME_JMS_ACTIVEMQ_BROKER_URL));
-
-        activeMQConnectionFactory.setNonBlockingRedelivery(true);
-
-        return activeMQConnectionFactory;
+    public JmsConfiguration dlmsRequestJmsConfiguration(final JmsConfigurationFactory jmsConfigurationFactory) {
```
Comment (hannek24) [inline osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java]: Creating such a bean which is exposed is not necessary.
This bean is only used in method dlmsRequestsMessageListenerContainer.

File: osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java [hunk 2]
```diff
@@ -134,278 +47,70 @@ public MessagingConfig() {
...
-        redeliveryPolicyMap.setDefaultEntry(this.defaultRedeliveryPolicy());
-        redeliveryPolicyMap.put(this.dlmsRequestsQueue(), this.dlmsRequestsRedeliveryPolicy());
-        redeliveryPolicyMap.put(this.dlmsResponsesQueue(), this.dlmsResponsesRedeliveryPolicy());
-        redeliveryPolicyMap.put(this.osgpRequestsQueue(), this.osgpRequestsRedeliveryPolicy());
-        redeliveryPolicyMap.put(this.osgpResponsesQueue(), this.osgpResponsesRedeliveryPolicy());
-        redeliveryPolicyMap.put(this.dlmsLogItemRequestsQueue(), this.dlmsLogItemRequestsRedeliveryPolicy());
-        return redeliveryPolicyMap;
+    public DefaultMessageListenerContainer dlmsRequestsMessageListenerContainer(
+            JmsConfiguration dlmsRequestJmsConfiguration) {
+        return dlmsRequestJmsConfiguration.getMessageListenerContainer();
```
Comment (hannek24) [inline osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java]: <code>
		return jmsConfigurationFactory.initializeConfiguration("jms.dlms.requests", this.dlmsRequestsMessageListener)
				.getMessageListenerContainer();
</code>

File: osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java (hunk 1 above)
Comment (rudidevries) [inline osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java]: We discussed not to do this, and changes were reverted.

File: osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java (hunk 2 above)
Comment (rudidevries) [inline osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/application/config/MessagingConfig.java]: We discussed not to do this, and changes were reverted.

**Pull request merged**